nautobot/extras/tests/test_jobs.py

```diff
@@ -176,3 +176,22 @@ def test_read_only_no_commit_field(self):
 <input class="form-control form-control" id="id_var" name="var" placeholder="None" required type="text">
 <br><span class="helptext">Hello</span><input id="id__commit" name="_commit" type="hidden" value="False"></td></tr>""",
             )
+
+
+class JobFileUploadTest(TestCase):
```

Comment:
Possibly need to add

```python
@skipIf(
    "dummy_plugin" not in settings.PLUGINS,
    "dummy_plugin not in settings.PLUGINS",
)
```

?

Reply:
Oh for sure. Thank you. I really wanted to knock this out last night but I hit a wall!